charts/kargo/templates/_helpers.tpl

```diff
@@ -102,22 +68,3 @@ app.kubernetes.io/component: webhooks-server
 {{- $template := index . 2 }}
 {{- include $template (dict "Chart" (dict "Name" $subchart) "Values" (index $dot.Values $subchart) "Release" $dot.Release "Capabilities" $dot.Capabilities) }}
 {{- end }}
-
-{{/*
-Return the appropriate apiVersion for a networking object.
-*/}}
-{{- define "networking.apiVersion" -}}
-{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
-{{- print "networking.k8s.io/v1" -}}
-{{- else -}}
-{{- print "networking.k8s.io/v1beta1" -}}
-{{- end -}}
-{{- end -}}
-
-{{- define "networking.apiVersion.isStable" -}}
-  {{- eq (include "networking.apiVersion" .) "networking.k8s.io/v1" -}}
-{{- end -}}
-
-{{- define "networking.apiVersion.supportIngressClassName" -}}
-  {{- semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion -}}
-{{- end -}}
```

Comment:
Good call removing this. When I added Ingress support in #540, I made a conscious decision that we didn't need to support versions of k8s this old. Thanks for cleaning this up.